agents_catalog/31-drug-label-automated-reasoning/README.md

```diff
@@ -0,0 +1,99 @@
+# Automated Reasoning for Drug Label Information
+
+## Summary
+
+This project provides a example of using Amazon Bedrock Automated Reasoning to defend against retrieval augmented generation (RAG) poisoning attacks. It uses a mock drug label based on Amazon basic care ibuprofen 200 mg tablets as the example data.
+
+## Getting Started
+
+### Prerequisities
+
+- AWS CLI configured with appropriate permissions
+- Python 3.13+ with [uv package manager](https://docs.astral.sh/uv/getting-started/installation)
+
+### Installation
+
+1. Clone this repository
+
+```bash
+git clone XXXX
+cd docs-ar-demo
+```
+
+1. Install Python dependencies
+
+```bash
+uv sync
+```
+
+### Create Automated Reasoning Policy
+
+![Create Automated Reasoning policy](static/create_policy.png)
+
+1. Open your web browser and navigate to the [Amazon Bedrock console](https://console.aws.amazon.com/bedrock).
+
+1. Navigate to **Build > Automated Reasoning**.
+
+1. Choose **Create policy**.
+
+1. In the **Policy details** section, give your policy a descriptive name, like "AnyDrug Drug Facts".
+
+1. In the **Source** section, choose **Enter text**.
+
+1. Copy the contents of `data/anydrug.md` and paste them into the **Source content** text field.
+
+1. Copy the following description and paste it into the **Describe the source content** text field:
+
+    ```bash
+    I'm building a chatbot that answers patient questions about the AnyDrug 200 mg tablets medication.
+    Below is an example question and answer:
+
+    - Question: Can I use anydrug to treat my back pain?
+    - Answer: Yes, AnyDrug is indicated for the relief of mild to moderate pain, including back pain.
+    ```
+
+1. Choose **Create policy** and wait 5-10 minutes for the Bedrock Automated Reasoning to finish importing the content.
+
+1. The new policy should have around 80 rules and variables. Choose **View definitions** to see more information about the policy.
+
+![Policy definitions](static/policy_definitions.png)
+
+1. Navigate to the **Policy details** tab and take note of the **Policy ID** value. You can also get this information using the AWS CLI and `jq` utility:
+
+```bash
+aws bedrock list-automated-reasoning-policies | jq '.automatedReasoningPolicySummaries | sort_by(.createdAt) | last'
+```
+
+### Deploy CloudFormation Stack
+
+1. (Optional) If needed, create an Amazon S3 bucket in your account with `aws s3 mb amzn-s3-demo-bucket`, replacing `amzn-s3-demo-bucket` with a globally unique bucket name.
```

Comment:
amzn* is a restricted keyword and cannot be used in s3 buckets, best to give another default suggestion